data/json/items/vehicle/engine.json

```
@@ -257,5 +257,38 @@
     "weight": 8000,
     "volume": 2,
     "price": 3500
+  },
+  {
+    "id": "small_turbine_engine",
+    "copy-from": "engine_gasoline",
+    "type": "ENGINE",
+    "name": "1350 hp gas turbine engine",
+    "description": "A gas turbine engine, usually used for military vehicles. Known for its high rate of fuel consumption.",
```

Need to use double spaces between sentences in description.
```suggestion
    "description": "A gas turbine engine, usually used for military vehicles.  Known for its high rate of fuel consumption.",
```

---

Spaces added, and I have switched them over to dropping fewer alloy plates rather than a larger number of alloy sheets.